front-end/src/app/home/components/social/social.component.scss

```diff
@@ -0,0 +1,19 @@
+ul {
+    list-style: none;
+}
+
+.iconBtn {
+    min-width: max-content;
+    min-height: 3.5rem;
+}
+
+.iconBtn:hover {
+    border-radius: 50%;
+    box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.25),
```

Comment:
Make sure that the radius is actually rendering a rounded rectangle, because usually using % in radius lead to more "ovale" shape than rounded corner

Reply:
c'est intentionnel